simpa_tests/automatic_tests/test_adapters_additional_mcx_flags.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import numpy as np
+
+from simpa import MCXAdapterReflectance, MCXAdapter, Tags, Settings
+
+class TestAdditionalFlagsMCXReflectanceAdapter(unittest.TestCase):
```

Comment:
This test looks good, I would add a general test case for the `SimulationModule` base class